tests/utils/test_db.py

```diff
@@ -0,0 +1,175 @@
+from unittest.mock import patch
+
+from page_analyzer.utils.db import (
+    create_check,
+    create_url,
+    get_url_by_name,
+    get_url_checks,
+    get_url_data,
+    get_urls,
+)
+
+
+def test_create_url(test_database: None):
+    # given
+    url_name = "https://example.com"
+
+    # when
+    url_id = create_url(url_name)
+
+    # then
+    created_url = get_url_data(url_id)
+    assert created_url["name"] == url_name
+    assert url_id == created_url["id"]
+
+
+def test_get_url_data(test_database: None):
+    # given
+    url_name = "https://example.com"
+
+    create_url(url_name)
+
+    # when
+    url_data_result = get_url_data(1)
+
+    # then
+    assert url_data_result["name"] == url_name
```

Comment:
tests should give a hint about what code returns, it's a bit hard to tell what else do we have in the `url_data_result` - let's compare `url_data_result` to a dict with actual data

note: you might want to use [freezegun](https://github.com/spulec/freezegun) to freeze time